.github/workflows/build.yml

```diff
@@ -55,12 +55,14 @@ jobs:
         id: get_branch
         run: echo ::set-output name=GIT_BRANCH::${GITHUB_REF##*/}
 
-      - name: Build and push Docker images
-        uses: docker/build-push-action@v1.1.0
+      - name: Login to DockerHub Registry
+        uses: docker/login-action@v1
         with:
           username: ${{ secrets.DOCKER_USER }}
           password: ${{ secrets.DOCKER_TOKEN }}
-          repository: squarescale/cli
-          tags: ${{ steps.get_branch.outputs.GIT_BRANCH }}
-          tag_with_ref: true
-          tag_with_sha: true
+
+      - name: Build and Push Docker Image
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: squarescale/cli:${{ steps.get_branch.outputs.GIT_BRANCH }}
```

Comment:
Just as a safety check before approving:

why did the tag_with_ref and tag_with_sha disappeared ?